src/Castle.Core.Tests/Castle.Core.Tests.csproj

```diff
@@ -59,11 +65,15 @@
 	<ItemGroup Condition="'$(TargetFramework)'=='netcoreapp2.1'OR'$(TargetFramework)'=='netcoreapp3.1'">
 		<PackageReference Include="System.Security.Permissions" Version="4.7.0" />
 	</ItemGroup>
+	<ItemGroup Condition="'$(TargetFramework)'=='net6.0'">
+		<PackageReference Include="System.Security.Permissions" Version="6.0.0" />
+	</ItemGroup>
```

Comment:
Is there a reason we can't just use 6.0.0 on all targets? Looks like it supports our targets.

Reply:
It does, but unfortunately the 6.0.0 package adds a target file for `netcoreapp2.0` which causes a build error (not just a warning) for EOL targets. See https://nuget.info/packages/System.Diagnostics.EventLog/6.0.0 -> `buildTransitive\netcoreapp2.0\System.Diagnostics.EventLog.targets`

Reply:
Cheers. All good.